### PR TITLE
[SCB-1603] Add the Getting Started for servicecenter, tookit, syncer to the website

### PR DESCRIPTION
**Pull request by ChinX**

### CODE REVIEW

**Comment by ChinX** [comment]
@WillemJiang I added some getting started documentation to the website, please review it.

File: _config.yml [hunk 1]
```diff
@@ -120,6 +120,9 @@ include:
   - _users
   - _faqs
   - _release
+  - _service-center
```
Comment (WillemJiang) [inline _config.yml]: It's better to use the "getstart" directory to host the files, as we could add more examples here..

Comment (ChinX) [reply _config.yml]: Next, I want to bring together the “user manual” and the “getting started”, so I want to divide the directory with the project name. Do you have any better suggestions?

Comment (WillemJiang) [reply _config.yml]: how about we setup the directory just like this "getStarted/service-center", "userManual/serivce-center"?  
In this way we can keep adding files according to the sub project name.

Comment (ChinX) [reply _config.yml]: Ok, thanks. I will do it with your suggestions.

Comment (ChinX) [reply _config.yml]: @WillemJiang Please review again, thanks!

**Pull request merged**